tests/unit/schemaloader.spec.js

```diff
@@ -1,14 +1,15 @@
 import { SchemaLoader } from '../../src/schemaloader'
+import * as sinon from 'sinon'
 
 describe('SchemaLoader', () => {
   let loader
   let fetchUrl
   let fileBase
 
   beforeEach(() => {
-    loader = new SchemaLoader()
```

Comment:
Is this a BC breaking change of the reworked SchemaLoder code?
Or should this be two tests, what is the difference when `ajax: true` - isn't it always AJAX?

Reply:
Divided the test code into cases where { ajax : true }  is required and cases where not.